src/Translator.php

```diff
@@ -75,30 +98,31 @@ public function getBasePath(): string
     /**
      * @param string|array $textToTranslate String of text to be translated or an array in case text has placeholders
      * @param string $domain Which translations domain to use
-     * @param string $context Context of the text (ex. urls, cli etc.)
-     * @param string|null $locale Target locale (to which language translate), if null, default locale will be used
+     * @param string|null $context Context of the text (ex. urls, cli etc.)
      *
      * @return string
-     * @throws GettextException
+     * @throws Exception\InvalidMoFileException
+     * @throws FileException
      */
     public function __(
         $textToTranslate,
         string $domain = self::DEFAULT_TRANSLATION_DOMAIN,
-        string $context = null,
-        string $locale = null
```

Comment:
There are cases when it's required to translate into locale different from the default one, do not remove this functionality.

Reply:
not anymore... Well, it is not possible to keep it with current changes, so we have to decide, do we want a feature that is not in use or we want some performance :)

Reply:
The feature is in use, otherwise I wouldn't mind to kick it out.

Reply:
Nope, this is not in use. It has only one place where this method is called and it doesn't pass any locale